project_variables.go

```diff
@@ -41,6 +41,7 @@ type ProjectVariable struct {
 	VariableType     VariableTypeValue `json:"variable_type"`
 	Protected        bool              `json:"protected"`
 	Masked           bool              `json:"masked"`
+	MaskedAndHidden  bool              `json:"masked_and_hidden"`
```

Comment:
```suggestion
	Masked  bool              `json:"masked"`
```

When reading the variable, you get the `masked` result back, not `masked_and_hidden` - Masked and Hidden have a value of `true` for this.